docs/storage/lumio/between-object-storages.md

```diff
@@ -0,0 +1,137 @@
+# Copying files directly between object storages
```

Comment:
"object storages" sounds weird to me. Could we rephrase this, "between object storage endpoints" for instance?